docs/source/upgrade_notes.rst

```diff
@@ -25,6 +25,13 @@ Upgrade Notes
   Both of those configuration options are disabled by default for now, but they will be enabled
   by default (with an option to opt-out) in a future major |st2| release.
 
+* The database schema for Mistral has changed. The executions_v2 table is no longer used. The
+  table is being broken down into workflow_executions_v2, task_executions_v2, and
+  action_executions_v2. After upgrade, using the Mistral commands from the command line such as
+  ``mistral execution-list`` will return an empty table. The records in executions_v2 have not
+  been deleted. The commands are reading from the new tables. There is currently no migration
+  script to move existing records from executions_v2 into the new tables.
```

Comment:
Should we add some note here on how people can still read from the old tables? e.g. something along the lines of "If you need to access the old tables, use `psql`"